autoload/sneak/util.vim

```diff
@@ -103,7 +103,8 @@ endf
 " Removes highlighting.
 func! sneak#util#removehl() abort
   silent! call matchdelete(w:sneak_hl_id)
-  silent! call matchdelete(w:sneak_sc_hl)
+  silent! call matchdelete(w:sneak_current_hl_id)
+  silent! call matchdelete(w:sneak_scope_hl_id)
```

Comment:
Wanted to make naming consistent here, and not think of 'smartcase' when reading 'sc'